tools/vsimporter/xib2nib/src/UIButton.h

```diff
@@ -17,14 +17,34 @@
 #pragma once
 #include "UIControl.h"
 
+struct EdgeInsets {
```

Comment:
exactly :(

---
In reply to: [107236214](https://github.com/Microsoft/WinObjC/pull/2269#discussion_r107236214) [](ancestors = 107236214)